newsView.test.js

```diff
@@ -0,0 +1,66 @@
+/**
+* @jest-environment jsdom
+*/
+
+const fs = require('fs');
+const { networkInterfaces } = require('os');
+const NewsView = require('./newsView')
+  
+describe ("News View", () => {
+  it("displays headlines in the browser", () => {
+    document.body.innerHTML = fs.readFileSync('./index.html');
+
+    const newsModel = { 
+      getHeadlines: () => ['testing', 'testing 2'],
+      getLinks: () => [],
+      getImages: () => []
+    }
+    const newsView = new NewsView(newsModel);
+  
+    newsView.displayNews();
+  
+    expect(document.querySelectorAll('div.headline').length).toBe(2);
```

Comment:
Maybe you can make these tests more specific, by adding assertions to verify that the headlines are the correct ones (they should be the values returned by our mocked model above) — same comment for the other tests below, for the other attributes